apps/docs/src/app/platform/component-docs/platform-menu-button/platform-menu-button-header/platform-menu-button-header.component.html

```diff
@@ -1,10 +1,17 @@
 <header>Menu Button</header>
 <description>
-    Menu Button works as Dropdown.
-    <br />
-    It selects from Menu whichs appears on Menu button click.
+    The menu button is used for triggering the display of a menu. Its display usually shows a "down arrow" icon to
```

Comment:
Is this right the sentence? `display of a menu` ? It does not sound right

Reply:
updated documentation and reviewed with team member.